### PR TITLE
Adds helps in the radio buttons + adds help for dissolved and intact

### LINKED ISSUE #8619: Write explanations for intact and dissolved
In the area of influence analysis we should explain what `intact` and `dissolved` means. I'll be adding a way to include help tooltips inside the radio button options… in the meantime, could you write a short explanation for each of those concepts?

### PR DESCRIPTION
**Pull request by javierarce**
This PR updates the radio editor of the forms to allow showing a help tooltip. The improvement was needed in the `area-of-influence` analysis to explain the `intact` and `dissolved` concepts.

To use this feature, simply add a new `help` element inside of the corresponding hash of the element you need to explain:

```
dissolved: {
  type: 'Radio',
  title: _t('editor.layers.analysis-form.boundaries'),
  options: [
    {
      val: 'false',
      label: _t('editor.layers.analysis-form.intact'),
      help: _t('editor.layers.analysis-form.intact-help')
    }, {
      val: 'true',
      label: _t('editor.layers.analysis-form.dissolved'),
      help: _t('editor.layers.analysis-form.dissolved-help')
    }
  ]
}
```

Since adding the help interrogation `(?)` used in other places took too much space, I've decided to underline the text labels like this:

screen shot 2016-08-11 at 11 51 07
screen shot 2016-08-11 at 11 51 13

CR: @xavijam 
CC: @saleiva 

Closes #8619

### CODE REVIEW

**Comment by piensaenpixel** [comment]
oh love it

File: lib/assets/javascripts/cartodb3/components/form-components/editors/radio.js [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
+    return this;
+  },
+
+  _setHelp: function () {
+    var containsHelp = _.find(this.schema.options, function (option) {
+      return option.help;
+    });
+
+    if (containsHelp) {
+      this._helpTooltip = this._createTooltip({
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/radio.js]: I have a doubt, will it create two tooltips or one?

**Comment by xavijam** [comment]
Apart from my question, all good man :es

File: lib/assets/javascripts/cartodb3/components/form-components/editors/radio.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/radio.js]: I'll create just one tooltip object.

**Pull request merged**